packages/plugins/i18n/admin/src/translations/en.json

```diff
@@ -14,11 +14,11 @@
   "Field.localized": "This value is unique for the selected locale",
   "Field.not-localized": "This value is common to all locales",
   "Settings.list.actions.add": "Add new locale",
-  "Settings.list.actions.delete": "Delete a locale",
+  "Settings.list.actions.delete": "Delete {name} locale",
   "Settings.list.actions.deleteAdditionalInfos": "This will delete the active locale versions <em>(from Internationalization)</em>",
   "Settings.list.actions.publishAdditionalInfos": "This will publish the active locale versions <em>(from Internationalization)</em>",
   "Settings.list.actions.unpublishAdditionalInfos": "This will unpublish the active locale versions <em>(from Internationalization)</em>",
-  "Settings.list.actions.edit": "Edit a locale",
+  "Settings.list.actions.edit": "Edit {name} locale",
```

Comment:
A bit nit picky but "Edit locale Fr" would read better to me than "Edit Fr
locale". wdyt? 🤷‍♂️

Reply:
You've made me doubt myself now lol, it'll normally read "Edit French (fr) locale" that's fine right? 😅

Reply:
Yeah your way is better 😆. Sorry I should've looked at the tests before commenting